.ci/docker/common/utils.sh

```diff
@@ -27,3 +27,20 @@ conda_run() {
 pip_install() {
   as_ci_user conda run -n "py_${PYTHON_VERSION}" pip install --progress-bar off "$@"
 }
+
+init_sccache() {
```

Comment:
np: why in the utils.sh file? can it be just inside install_cache.sh file directory?